xbmc/settings/AdvancedSettings.cpp

```diff
@@ -109,7 +109,7 @@ void CAdvancedSettings::Initialize()
 
   m_audioHeadRoom = 0;
   m_ac3Gain = 12.0f;
-  m_audioApplyDrc = true;
+  m_audioApplyDrc = -1.0f;
```

Comment:
we provide various levels for that purpose, don't we? but we can do this in a separate pr. no need to block this one